masonry/src/widget/mod.rs

```diff
@@ -52,7 +52,7 @@ pub use widget_state::WidgetState;
 
 pub use sized_box::BackgroundBrush;
 #[doc(hidden)]
-pub use widget::{Widget, WidgetId};
+pub use widget::{AllowRawMut, Widget, WidgetId};
```

Comment:
I don't think these should be re-exported here. Instead, we should make `widget::widget` be `pub(crate)`, then re-export from that module in the root.

That is morally equivalent to making this export item `pub(crate)`, although there's an inaccurate diagnostic if you actually try and do that.